specifications/api/swagger.yaml

```diff
@@ -4085,6 +4085,13 @@ x-parametersMaxCount: &parametersMaxCount
     type: integer
     minimum: 1
 
+x-parametersAddress: &parametersAddress
+  in: query
+  name: address
+  description: |
+    An optional address.
```

Comment:
Could you specify in more detail what the endpoint does when this optional parameter is supplied? Something along the lines of

```
An optional address.
If given, the list of transactions will be filtered
to only contain transactions that refer to this address in their inputs or outputs.
This may exclude transactions whose inputs reference outside transactions
that are not part of the transaction history of the wallet.
```

Reply:
thanks! Excellent suggestion -> done

Reply:
We have decided that we have to write specifications for any user-facing change. 🤓